src/main/java/com/volunteer/enterprise/VolunteerServiceStub.java

```diff
@@ -1,4 +1,25 @@
 package com.volunteer.enterprise;
 
-public class VolunteerServiceStub {
+
+import dto.Volunteer;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class VolunteerServiceStub implements VolunteerService {
+    @Override
+    public void save(Volunteer volunteer) {
+
+    }
+
+    @Override
+    public void delete(Integer i) {
+
+    }
+
+    @Override
+    public List<VolunteerService> fetchAll() {
+        return null;
+    }
```

Comment:
Only features that meet the definition of done should be merged to master.  This is an empty class, with empty methods, so it is clutter.  That increases technical debt.  A code review should focus on decreasing technical debt of existing code, more so than adding new features, which may introduce more technical debt.